bundle/Core/SignalSlot/AbstractPublishSlot.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+namespace Netgen\TagsBundle\Core\SignalSlot;
```

Comment:
Can you move all these slots to a separate namespace, e.g. `Netgen\TagsBundle\Core\SignalSlot\HttpCache`, so they do not clash with potential future slots?